test/GSteps.cpp: -Werror build failure on ignored-qualifiers

Having cloned recursively, using gcc 8.3.0, I get the following build failure:

````Scanning dependencies of target test_GSteps
[ 75%] Building CXX object CMakeFiles/test_GSteps.dir/test/GSteps.cpp.o
In file included from /home/jamespharvey20/home/code/gunit.recursive.git/libs/googletest/googletest/include/gtest/gtest.h:58,
                 from /home/jamespharvey20/home/code/gunit.recursive.git/include/GUnit/GSteps.h:10,
                 from /home/jamespharvey20/home/code/gunit.recursive.git/test/GSteps.cpp:8:
/home/jamespharvey20/home/code/gunit.recursive.git/test/GSteps.cpp: In member function ‘void testing::v1::GTEST<testing::v1::detail::string<'\"', 'C', 'o', 'n', 'v', 'e', 'r', 't', 'i',
 'b', 'l', 'e', '\"', '\000'>, testing::v1::detail::string<> >::TestBodyImpl(testing::v1::detail::TestRun&)’:
/home/jamespharvey20/home/code/gunit.recursive.git/test/GSteps.cpp:71:79: error: type qualifiers ignored on cast result type [-Werror=ignored-qualifiers]
     EXPECT_FALSE(static_cast<const bool>(detail::Convertible<std::string>{"0"}));
                                                                               ^
cc1plus: all warnings being treated as errors
make[2]: *** [CMakeFiles/test_GSteps.dir/build.make:63: CMakeFiles/test_GSteps.dir/test/GSteps.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:748: CMakeFiles/test_GSteps.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
````

--- a/test/GSteps.cpp
+++ b/test/GSteps.cpp
@@ -68,7 +68,7 @@ GTEST("Convertible") {
 
     const bool c = detail::Convertible<std::string>{"true"};
     EXPECT_TRUE(c);
-    EXPECT_FALSE(static_cast<const bool>(detail::Convertible<std::string>{"0"}));
+    EXPECT_FALSE(static_cast<bool>(detail::Convertible<std::string>{"0"}));
   }
 }
 

